docs/development/best-practices.md

```diff
@@ -3,6 +3,26 @@
 This document explains our best practices.
 Follow these best practices when you're working on our code.
 
+## Git branch names
```

Comment:
I think it's a good idea to put this section near the top, as that way it's the first thing you see before you start creating branches. But if you want I can move it someplace else.